gradle/dependencies.gradle

```diff
@@ -13,16 +13,16 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '119.0.1'
+      mapboxNavigatorVersion = '121.0.0'
   }
   println("Navigation Native version: " + mapboxNavigatorVersion)
 
   version = [
-      mapboxMapSdk              : '10.9.0',
+      mapboxMapSdk              : '10.10.0-beta.1',
```

Comment:
It will be beta.2, right?

Reply:
Latest is https://github.com/mapbox/mapbox-maps-android/releases/tag/v10.10.0-beta.1.

Reply:
We're releasing beta.2 with payment_methods right now.

Reply:
Your original comment points to Maps SDK 🙃 I understand you mean `mapbox-java`.

Reply:
Oooh, right, sorry, I misread that.

Reply:
yes, mapbox-java will be 6.10.0-beta.2